packages/arb-validator-core/gobridge/inboxTopChallenge.go

```diff
@@ -70,14 +73,14 @@ func (c *inboxTopChallenge) Bisect(
 	c.commitToSegment(hashes)
 	c.asserterResponded()
 
-	c.client.pubMsg(c.challengeData, arbbridge.MaybeEvent{
+	c.client.pubMsg(c.challenge, arbbridge.MaybeEvent{
 		Event: arbbridge.InboxTopBisectionEvent{
 			ChainInfo: arbbridge.ChainInfo{
 				BlockId: c.client.getCurrentBlock(),
 			},
 			ChainHashes: chainHashes,
 			TotalLength: chainLength,
-			Deadline:    c.client.challenges[c.contractAddress].deadline,
+			Deadline:    c.client.challenges[c.contractAddress].challengeData.deadline,
```

Comment:
Unnecessary indirection to get `challengeData`

Reply:
fixed